ros/resources/v0/hosts.py

```diff
@@ -5,9 +5,14 @@
 from ros.models import PerformanceProfile
 from ros.utils import is_valid_uuid
 from ros.lib.host_inventory_interface import fetch_all_hosts_from_inventory
+from ros.api.common.pagination import build_paginated_system_list_response
+
+MAX_HOSTS_PER_REP = 1000
```

Comment:
@kgaikwad outputting 1000 hosts per request seems to be very huge. Can we set the default limit to 10? 
Also, I think the variable name should be `DEFAULT_HOSTS_PER_REP`, As you are using it to set the `limit` value when `limit` is not passed in query params.

Reply:
makes sense :+1: